### DISCUSSION TITLE
Please tell me your favorite Anime!

### DISCUSSION THREAD

**Discussion by hirominn** [NAIST-SE/AttractingContributors#127] [General]
https://github.com/hirominn/AttractingContributors

**Comment by zhangyongxin1212** [comment]
https://github.com/zhangyongxin1212/AttractingContributors-Game/pulls
added and plz help me

**Reply by hirominn** [reply]
Thanks! Of course, I will!

**Comment by nagata-atsuki** [comment]
added
Please help me
https://github.com/nagata-atsuki/AttractingContributors-Athlete

**Reply by hirominn** [reply]
Thanks! I will!

**Comment by t-matsumoto4758** [comment]
Added! 
please help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Reply by hirominn** [reply]
Thanks! Got it!

**Comment by veekay7** [comment]
I have added to your repo.
Please help me. Cheers.
https://github.com/VantelinLuca/AttractingContributors-WishCountry

**Reply by hirominn** [reply]
Good taste! Of course, I will!

**Comment by hernessteve** [comment]
Kill la kill one of the best ever! I recently installed few new addons on my firestick  and found pretty cool channels with good choice of anime there. Must admit that jailbroken firestick opened for me a huge library of new content. If someone still doesn't know what i am talking about - [https://www.firesticktricks.com/jailbroken-firestick.html](https://www.firesticktricks.com/jailbroken-firestick.html) that't the good source to read about it.
